src/IdBrokerClient.php

```diff
@@ -3,13 +3,29 @@
 
 use Exception;
 use GuzzleHttp\Command\Result;
+use IPBlock;
 
 /**
  * IdP ID Broker API client implemented with Guzzle.
  */
 class IdBrokerClient extends BaseClient
 {
     /**
+     * The key for the constructor's config parameter that refers
+     * to the trusted IP ranges.
+     */
+    const TRUSTED_IPS_CONFIG = 'trusted_ip_ranges';
+
+    /**
+     * The list of trusted IP address ranges (aka. blocks).
+     *
+     * @var IPBlock[]
+     */
+    private $trustedIpRanges = [];
+
```

Comment:
I'd like another boolean config for `$assertValidBrokerAddress` and if it is true, then ensure `$trustedIpRanges` is not empty and the resolved address is valid in the ranges. The reason I want the boolean is it helps prevent typos from disabling this feature. I'll be passing the list of ranges to the configuration dynamically in Terraform, and if I make a mistake or something changes in there it could be an empty list which would essentially silently disable this feature.